tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -11430,12 +11430,12 @@ void ValidatePatchConstantFunctionsExist(clang::Sema *self) {
 // should exist on this shader stage
 void DiagnoseEntryAttrAllowedOnStage(clang::Sema *self,
                                      FunctionDecl *entryPointDecl,
-                                     DXIL::ShaderKind shaderKind) {
+                                     DXIL::ShaderKind shaderKind,
+                                     bool isActiveEntry) {
```

Comment:
What's the point of adding `isActiveEntry` now?  It isn't used, so I don't think we should add it until it has a real purpose.